### ISSUE TITLE
Path resolving within sass libraries

### ISSUE THREAD

**Issue by sveetch** [dahlia/libsass-python#114]
I'm trying to compile [Foundation5](http://foundation.zurb.com/sites/docs/v/5.5.3/sass.html) from its sass sources and it fails to resolve some relative path inside the library.

I used to import the Foundation5 as a sass library and so used sassc like this:

```
sassc --include-path=../foundation5/bower_components/foundation/scss/ scss/app.scss ../project/webapp_statics/css/app.css
```

As i see, some components try to import some other components using a path relative to the current sass lib file:

```
@import 'global';
```

or 

```
@import '../functions';
```

Where "global" is in the same directory that the current sass lib file, and "functions" is in parent directory.

But if i edit Foundation sass lib files to use a "full" relative path from the lib directory, it works:

```
@import 'foundation/components/global';
```

It troubles me because Foundation5 is certified to compile with Compass and also with the node.js wrapper for libsass, so i'm guessing what is the standard behavior for import path resolving within sass libraries ?

**Comment by asottile** [comment]
I'm having trouble reproducing this:

```
$ bower install foundation
...
$ echo "@import 'foundation';" > test.scss
$ sassc --version
sassc 0.10.0
$ sassc test.scss test.css --include-path=bower_components/foundation/scss
$ ls test.css
test.css
```

Could you provide some more information?
- Contents of application scss
- Version
- Error messages

**Comment by sveetch** [comment]
So i tried your process and it works. Then i tried to move the foundation scss stuff out of the current directory and compile a main scss in a subdirectory to reproduce something like my file structure:

```
$ bower install foundation
$ mv bower_components ../
$ echo "@import 'foundation';" > scss/test.scss
$ sassc scss/test.scss test.css --include-path=../bower_components/foundation/scss/
sassc: error: Error: File to import not found or unreadable: global
       Parent style sheet: ../bower_components/foundation/scss/foundation/components/_grid.scss
        on line 5 of ../bower_components/foundation/scss/foundation/components/_grid.scss
>> @import 'global';
   ^
```

I tried also to use the bower_components path with the test.scss in the current directory and it fails like my previous sample.

Finally i tried to compile the scss/tests.scss using an absolute path for the included lib and it works:

```
$ sassc scss/test.scss test.css --include-path=/home/emencia/projects/pims/try-libsass/bower_components/foundation/scss/
```

So using a relative include path out of the current directory is troubling sassc for imports ? is it the standard behavior ?

**Comment by asottile** [comment]
fwiw I can also reproduce it with node-sass, which means it is a bug with libsass itself:

```
# sassc here refers to the python binding (we should really rename it to reduce confusion)
$ sassc --include-path ../bower_components/foundation/scss/ test.scss test.css
sassc: error: Error: File to import not found or unreadable: global
       Parent style sheet: ../bower_components/foundation/scss/foundation/components/_grid.scss
        on line 5 of ../bower_components/foundation/scss/foundation/components/_grid.scss
>> @import 'global';
   ^

$ node-sass --include-path ../bower_components/foundation/scss/ test.scss test.css
{
  "formatted": "Error: File to import not found or unreadable: global\n       Parent style sheet: ../bower_components/foundation/scss/foundation/components/_grid.scss\n        on line 5 of ../bower_components/foundation/scss/foundation/components/_grid.scss\n>> @import 'global';\n   ^\n",
  "message": "File to import not found or unreadable: global\nParent style sheet: ../bower_components/foundation/scss/foundation/components/_grid.scss",
  "column": 1,
  "line": 5,
  "file": "../bower_components/foundation/scss/foundation/components/_grid.scss",
  "status": 1
}
```

**Comment by asottile** [comment]
Created https://github.com/sass/libsass/issues/1846 in the upstream project, let's track things there.

**Comment by sveetch** [comment]
Ok, I imagine the Foundation stuff has some grunt task that may change current working dir or something else to succeed the compile and that's why i thought it was not a "standard" behavior and was specific libsass-python bug.

I'm watching the related issue, until something change it's just a matter of changing some path.

Thanks.

**Comment by amarsahinovic** [comment]
Version 3.3.3 that fixes the referenced issue is out https://github.com/sass/libsass/releases/tag/3.3.3

**Comment by asottile** [comment]
Upstream 3.3.3 is now available in our version 0.10.1
